src/main/java/chat/task/Todo.java

```diff
@@ -62,13 +62,13 @@ public static Todo createTodo (String str) throws ChatException {
      * @return Comma separated string with all parameters listed out.
      */
     public String allParameterStr() { 
-        return String.format("T,%s,%s", this.getDone(), this.getName());
+        return String.format("T,%s,%s", this.getIsDone(), this.getName());
     }
 
     /**
      * Returns a string that shows the details of the task.
-     * <p>[ ] will be displayed if done = false.</p>
-     * <p>[X] will be displayed if done = true.</p>
+     * <p>[ ] will be displayed if isDone = false.</p>
+     * <p>[X] will be displayed if isDone = true.</p>
```

Comment:
I feel dont need to so explicitly show what will be returned since the toString method is quite clear